### PR TITLE
refactor(base-cluster): deleting everything with deadMansSwitch

### PR DESCRIPTION
**Pull request by CrocodileHunter8**
Deleting everything with deadManSwitch
@tasches did i do everything right in regard of the version numbers + the change of the version number in the 4allportal helm chart?

For Ticket: CL-45:
https://jira.4allportal.net/browse/CL-45

### CODE REVIEW

File: charts/4allportal/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "3.10.35"
 description: A Helm chart for 4ALLPORTAL version 3.10.0 and up
 name: 4allportal
-version: 19.0.42
+version: 19.0.43
```
Comment (tasches) [inline charts/4allportal/Chart.yaml:5]: you haven't changed anything in the 4allportal chart so no version bump necessary

File: charts/base-cluster/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A generic, base cluster setup
 name: base-cluster
-version: 37.0.6
+version: 37.0.7
```
Comment (tasches) [inline charts/base-cluster/Chart.yaml:4]: you have removed a functionality of the base-cluster chart which is considered a breaking change.
The have to increase the major and not the patch version

**Review by tasches — CHANGES_REQUESTED** [review]

**Comment by tasches** [comment]
Error in values.schema. See pipeline. Fix necessary

File: charts/base-cluster/values.schema.json [hunk 3]
```diff
@@ -837,22 +837,7 @@
...
-            "apiKey": {
-              "type": "string"
-            },
-            "pingKey": {
-              "type": "string"
-            }
-          },
-          "additionalProperties": false
-        }
+        },
```
Comment (tasches) [inline charts/base-cluster/values.schema.json:840]: still errors in the schema.

**Review by tasches — CHANGES_REQUESTED** [review]

**Review by tasches — CHANGES_REQUESTED** [review]
No documentation was added. Because this is a major release and there are breaking changes due to the removal of the deadmanswitch. You need a documentation. You can try it yourself. Use an old values file with deadmanswitch enbaled and try the update. It will fail. See values.schema

**Review by tasches — CHANGES_REQUESTED** [review]

File: charts/base-cluster/README.md [hunk 4]
```diff
@@ -323,3 +320,7 @@ This update removes the old security scanner estafette and installs aquasecuriti
 ### To 37.1.6
 
 You can now add an email configuration for the alertmanager. If your email server uses port 456 SMARTTLS will be disabled automaticaly. It is also possible to add custome routes for the alertmanager. For the syntax please refer to the alertmanager [documentation](https://prometheus.io/docs/alerting/latest/configuration/) or our values.schema.json.
+
+### To 38.0.0
+
+This release removes everything related to the deadMansSwitch function
```
[the remark this answers is not in the document]
Comment (CrocodileHunter8) [reply charts/base-cluster/README.md]: i add it

**Comment by tasches** [comment]
rejected

**Pull request closed without merging**